dotnet/src/webdriver/Internal/NullableAttributes.cs

```diff
@@ -0,0 +1,149 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We should put a link here where we stole it.

Reply:
Resolved on Slack to avoid adding our own nullable attributes.